<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QUaModbusClientWidget</class>
 <widget class="QWidget" name="QUaModbusClientWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>367</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <widget class="QPushButton" name="pushButtonConnect">
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="pushButtonPerms">
     <property name="text">
      <string>Permissions</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="Line" name="line1">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="5">
    <widget class="QUaModbusClientWidgetStatus" name="widgetClientStatus" native="true"/>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="pushButtonDelete">
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>175</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer_1">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>175</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="5">
    <widget class="QUaModbusClientWidgetEdit" name="widgetClientEdit" native="true"/>
   </item>
   <item row="6" column="4">
    <widget class="QPushButton" name="pushButtonApply">
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="5">
    <widget class="Line" name="line3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="5">
    <widget class="Line" name="line2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="pushButtonAddBlock">
     <property name="text">
      <string>Add Block</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="pushButtonClear">
     <property name="text">
      <string>Clear Blocks</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QUaModbusClientWidgetEdit</class>
   <extends>QWidget</extends>
   <header location="global">QUaModbusClientWidgetEdit</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QUaModbusClientWidgetStatus</class>
   <extends>QWidget</extends>
   <header location="global">QUaModbusClientWidgetStatus</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
